test/runtests.jl

```diff
@@ -1,18 +1,32 @@
 using Conda, Compat, VersionParsing
+using Compat
+using Compat: @info
 using Compat.Test
 
+exe = Compat.Sys.iswindows() ? ".exe" : ""
+
 Conda.update()
 
 env = :test_conda_jl
 @test Conda.exists("curl", env)
 Conda.add("curl", env)
 
+@testset "Install Python package" begin
+    Conda.add("python", env)
+    pythonpath = joinpath(Conda.python_dir(env), "python" * exe)
+    @test isfile(pythonpath)
+
+    cmd = Conda._set_conda_env(`$pythonpath -c "import zmq"`, env)
+    @test_throws Exception run(cmd)
```

Comment:
This won't work if `pyzmq` is already installed — people don't necessarily run `test Conda` on a clean installation.

Reply:
In particular, what happens if you run the tests twice in a row?  Won't `pyzmq` still be installed from the previous run?

Reply:
Oops, `@test !success(cmd)` was just a leftover from the previous version.

To make `]test Conda` easier, how about doing

```
try
    Conda.rm("--all", env)
catch
end
```

(or with something similar with cleaner handling of exception)?